mods/ts/rules/defaults.yaml

```diff
@@ -887,9 +887,11 @@
 		DamageInterval: 16
 		DamageTypes: BulletDeath
 		Terrain: Veins
-	GrantConditionOnTerrain@VEINS:
+	LeavesTrails@VEINS:
+		Image: veins
+		Palette: player
 		TerrainTypes: Veins
-		Condition: veins
-	WithIdleOverlay@VEINS:
-		Sequence: veins
-		RequiresCondition: veins
+		TrailWhileStationary: true
+		StationaryInterval: 16
+		MovingInterval: 16
```

Comment:
TBH I tried to maintain sync with the damage delivery but that already breaks at movestart with the starting delay (maybe I shoulda drop a commit to use -1 to not overwrite the delay with whatever value ticks had when switching). Guess the issue existed in TS as well.